### ISSUE TITLE
[Bug]: ERNIE Layout: target_size in preprocessing missing in call to _str2im()

### ISSUE THREAD

**Issue by waynemystir** [PaddlePaddle/PaddleNLP#4798]
In utils.py, the call `_str2im()` doesn't pass the given arg target_size. But the call to `_scale_same_as_image` does pass the target_size. This appears to create difference scales between the image size and bounding bound coordinates.

https://github.com/PaddlePaddle/PaddleNLP/blob/develop/model_zoo/ernie-layout/utils.py#L245
https://github.com/PaddlePaddle/PaddleNLP/blob/develop/model_zoo/ernie-layout/utils.py#L252-L257

**Comment by linjieccc** [comment]
@waynemystir Hi, thanks for your interest on ernie-layout!

At both pre-train and fine-tuning stages, images and boxes are normalized to the target size of 224 and 1000 respectively.

For image processing, please refer to here
https://github.com/PaddlePaddle/PaddleNLP/blob/develop/model_zoo/ernie-layout/utils.py#L114

**Comment by waynemystir** [comment]
Thank you @linjieccc . So the ERNIE Layout model expects the images and boxes to be normalized to different scales then?

**Comment by linjieccc** [comment]
Yes. The image is [224, 224], and the box is [1000, 1000]. Scale mapping relations can be learned by the model. Large images will increase memory usage while there is no improvement for the result.

**Comment by waynemystir** [comment]
Interesting. Thank you very much for this explanation @linjieccc !
